lib/challenge.js

```diff
@@ -19,6 +19,16 @@
   // - park
 
 // Answer Starts Here
+ //"calendar"
+// Description: our program keeps track of daily, weekly, monthly things.
+// Every "calendar" has
+// - a day
+ //- a week
+ //- a month
+ //Every "calendar" can
+// - tell what the current day is
+ //- what holidays are in the month
+ //- show tasks by date
```

Comment:
This is a reasonable abstraction, but we were expecting three. Can you think of two others that would make sense for this application?